ProjFS.Mac/PrjFSKext/KextLog.cpp

```diff
@@ -81,7 +81,13 @@ void KextLog_Printf(KextLog_Level loglevel, const char* fmt, ...)
     va_start(args, fmt);
     int messageLength = vsnprintf(message.logString, sizeof(message.logString), fmt, args);
     va_end (args);
-    int messageSize = sizeof(KextLog_MessageHeader) + messageLength + 1 /* null terminator */;
+    uint32_t messageSize;
+    if (__builtin_uadd_overflow(sizeof(KextLog_MessageHeader), messageLength, &messageSize) ||
+        __builtin_uadd_overflow(messageSize, 1, &messageSize))
```

Comment:
Here we'd skip logging a message to Kext.  Is it worth trying to truncate and write something or an error?

Reply:
Talked to @jrbriggs offline and that would be a change for a separate PR.